src/styles/mandybrigwell.js

```diff
@@ -57,8 +57,8 @@ export default class DemoStyle extends Style {
 		this._p5.noStroke();
 		
 		// We're going to jiggle the tiles a little with a slight random value
-		this.randomMinimum = 1-fxrand()*0.025;
-		this.randomMaximum = 1+fxrand()*0.025;
+		this.randomMinimum = 1-this._p5.random()*0.025;
```

Comment:
It change nothing, but fyi, you can use `this._p5.random(0.025)` instead of `this._p5.random()*0.025`

Reply:
Thanks. I got used to using the fxrand() which, if I'm correct, doesn't take a variable.

Reply:
yes! It's perfectly correct to use fxrand() usually. It's just specific to our project :)